test/e2e/framework/util.go

```diff
@@ -2406,11 +2406,19 @@ func WaitForStableCluster(c clientset.Interface, masterNodes sets.String) int {
 	}
 	allPods.Items = currentPods
 	scheduledPods, currentlyNotScheduledPods := e2epod.GetPodsScheduled(masterNodes, allPods)
-	for len(currentlyNotScheduledPods) != 0 {
+	systemPods, err := c.CoreV1().Pods(metav1.NamespaceSystem).List(metav1.ListOptions{})
+	ExpectNoError(err, "listing all pods in kube-system namespace while waiting for stable cluster")
+	_, currentlyNotScheduledSystemPods := e2epod.GetPodsScheduled(masterNodes, systemPods)
```

Comment:
scheduledSystemPods (and return that at the end)

Reply:
updated